SConstruct

```diff
@@ -59,42 +67,56 @@ platform = sys.platform
 msvc12 = GetOption('msvc12')
 sdk = GetOption('sdk')
 samples = GetOption('samples')
+qt_dir = GetOption('qt')
+
+if platform == 'win32':
+    from build_config_win32 import *
+elif platform == 'linux2':
+    from build_config_linux import *
 
-# TODO: move to build_config_<platform>
 if isX86:
-  if platform == 'win32':
-    target_arch = "x86"
-    arch_suffix = ''
-    win_def = 'WIN32'
-  else:
-    target_arch = 'i386'
+    arch = '32'
 else:
-  if platform == 'win32':
-    target_arch = "amd64"
-    arch_suffix = '_64'
-    win_def = 'WIN64'
-  else:
-    target_arch = 'x86_64'
+    arch = '64'
```

Comment:
Instead of this two-step process (of getting a boolean option and then creating a variable), can we use the variables directly? For example build_arch = GetOption('build_arch')

Reply:
Good catch